textblob/translate.py

```diff
@@ -44,11 +44,13 @@ def translate(self, source, from_lang='auto', to_lang='en', host=None, type_=Non
         if PY2:
             source = source.encode('utf-8')
         data = {"q": source}
-        url = u'{url}&sl={from_lang}&tl={to_lang}&hl={to_lang}&tk={tk}'.format(
+        url = u'{url}&sl={from_lang}&tl={to_lang}&hl={to_lang}&tk={tk}&client={client}&format={format}'.format(
             url=self.url,
             from_lang=from_lang,
             to_lang=to_lang,
             tk=_calculate_tk(source),
+            client="te",
+            format="html",
```

Comment:
This line is not needed. Actually, it seems that it can even lead to some problems:

With html as format: 
![image](https://user-images.githubusercontent.com/61357777/135761853-6ab19581-3c43-496f-ae49-9d4968cb6fb0.png)

Without it:
![image](https://user-images.githubusercontent.com/61357777/135761862-d0dac1bb-55c1-4b80-b305-c6d56382c35c.png)

So as it should, it returns the content in html format, which can cause issues when using with print as seen above. Therefore I suggest that this line should be omitted.

Changing client alone solves the issue #395.

Reply:
Cool! Thanks for the review! Already pushed changes. And fixed method "detect"